components/deck/build-deck.webc

```diff
@@ -226,3 +226,201 @@
   }
 }
 </style>
+
+<style webc:bucket="slides-theme">
+  @layer slide.theme {
+
+    @layer props {
+
+      /* Recursive Static Fonts */
+      @font-face {
+        font-display: auto;
+        font-family: 'Recursive Sans Linear';
+        font-stretch: normal;
+        font-style: normal;
+        font-weight: normal;
+        src: url('/fonts/RecursiveSansLnrSt-Regular.woff2') format('woff2');
+      }
+
+      @font-face {
+        font-display: auto;
+        font-family: 'Recursive Sans Linear';
+        font-stretch: normal;
+        font-style: normal;
+        font-weight: 700;
+        src: url('/fonts/RecursiveSansLnrSt-Bold.woff2') format('woff2');
+      }
+
+      @font-face {
+        font-display: auto;
+        font-family: 'Recursive Sans Linear';
+        font-stretch: normal;
+        font-style: italic;
+        font-weight: normal;
+        src: url('/fonts/RecursiveSansLnrSt-Italic.woff2') format('woff2');
+      }
+
+      @font-face {
+        font-display: auto;
+        font-family: 'Recursive Mono Linear';
+        font-stretch: normal;
+        font-style: normal;
+        font-weight: normal;
+        src: url('/fonts/RecursiveMonoLnrSt-Regular.woff2') format('woff2');
+      }
+
+      @font-face {
+        font-display: auto;
+        font-family: 'Recursive Mono Linear';
+        font-stretch: normal;
+        font-style: normal;
+        font-weight: 700;
+        src: url('/fonts/RecursiveMonoLnrSt-Bold.woff2') format('woff2');
+      }
+
+      @font-face {
+        font-display: auto;
+        font-family: 'Recursive Mono Linear';
+        font-stretch: normal;
+        font-style: italic;
+        font-weight: normal;
+        src: url('/fonts/RecursiveMonoLnrSt-Italic.woff2') format('woff2');
+      }
+    }
+
+    html {
+
+      /* background colors */
+      --bg: hsl(200 12% 95.098%);
+      --callout: hsl(193, 28%, 87%);
+      --mark-bg: var(--callout);
+
+      /* content colors */
+      --text: hsl(200deg 12% 4.902%);
+      --text-light: hsl(192.86deg 10% 27.451%);
+      --accent-color: hsl(330.2deg 67.742% 42.549%);
+      --border-color: hsl(195deg 10% 50%);
+      --shadow: hsl(195deg 30.769% 94.902% / 0.4);
+      --action: hsl(195deg 53% 27%);
+      --active: hsl(330deg 52.857% 27.451%);
+      --highlight: hsl(330deg 52.857% 27.451%);
+      --feature: hsl(195.65deg 45.098% 20%);
+    }
+  }
+
+  @layer base {
+    html {
```

Comment:
```suggestion
```

I think everything in the `slides-theme` bucket should probably be in the `slide.theme` layer (or sublayers, if needed).